templates/admin/library_stat_detail.html

```diff
@@ -60,14 +55,7 @@ <h3 class="my-2">
               <hr>
             {% endifchanged %}
             <div class="flex gap-x-1">
-              {% if item.commit__author__avatar_url %}
-                <img src="{{ item.commit__author__avatar_url }}"
-                     alt="github avatar"
-                     class="w-8 rounded">
-              {% else %}
-                <div class="w-8 h-8 rounded bg-silver">
-                </div>
-              {% endif %}
+              {% avatar name=item.commit__author__name image_url=item.commit__author__avatar_url href=None %}
```

Comment:
Why does "href" have different values here versus the previous one? Why is image_url different here from the previous one?

Reply:
Because the object is different.  Each item in this queryset is actually a `LibraryVersion` with annotations to count commits by commit author per version.